Replace Slack with Discord (#40751)
sourcegraph/sourcegraph: doc/dev/setup/quickstart.md

This is the quickstart guide for [developing Sourcegraph](../index.md).

> > NOTE: If you run into any troubles, you can alternatively consult the [deprecated quickstart instructions without `sg`](deprecated_quickstart.md) or reach out on Discord:

Reviewer: ```suggestion
> NOTE: If you run into any troubles, you can alternatively consult the [deprecated quickstart instructions without `sg`](deprecated_quickstart.md) or reach out for help:
```

Reviewer: One link is still Slack (the Sourcegraph team one)
